llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1236,20 +1235,22 @@ class LoopVectorizationCostModel {
   /// the loop.
   void collectInstsToScalarize(ElementCount VF);
 
-  /// Collect Uniform and Scalar values for the given \p VF.
+  /// Collect values that will not be widened, including Uniforms, Scalars, and
+  /// Instructions to Scalarize for the given \p VF.
   /// The sets depend on CM decision for Load/Store instructions
   /// that may be vectorized as interleave, gather-scatter or scalarized.
   /// Also make a decision on what to do about call instructions in the loop
   /// at that VF -- scalarize, call a known vector routine, or call a
   /// vector intrinsic.
-  void collectUniformsAndScalars(ElementCount VF) {
+  void collectNonVectorizedSetWideningDecisions(ElementCount VF) {
```

Comment:
thanks for the update, might be good to sprinkle an `And` for slightly easier readability
```suggestion
  void collectNonVectorizedAndSetWideningDecisions(ElementCount VF) {
```